source/includes/example-connect.rst

```diff
@@ -50,3 +50,23 @@ following command on one line:
                        clusterTwo02.fancyCorp.com:20020,
                        clusterTwo03.fancyCorp.com:20020
 
+To use ``mongosync`` with Atlas clusters, you add the
+:urioption:`ssl=true <ssl>` option to enable SSL. For example:
+
+.. code-block:: shell
+
+   mongosync \
+      --cluster0 'mongodb://clusterAdmin:superSecret@clusterOne01.fancyCorp.com:20020,clusterOne02.fancyCorp.com:20020,clusterOne03.fancyCorp.com:20020/?ssl=true' \
+      --cluster1 'mongodb://clusterAdmin:superSecret@clusterTwo01.fancyCorp.com:20020,clusterTwo02.fancyCorp.com:20020,clusterTwo03.fancyCorp.com:20020/?ssl=true'
+
+You can also use ``mongodb+srv`` connection strings with ``mongosync``.
```

Comment:
Thought: Does it make sense to reinforce the ssl usage distinction with a line like, "You do not need to add the :urioption:`ssl=true <ssl>` option to a `mongodb+srv`` connection string."

Reply:
Done.